app/assets/javascripts/controllers/ems_common/ems_common_form_controller.js

```diff
@@ -194,6 +194,7 @@ ManageIQ.angular.app.controller('emsCommonFormController', ['$http', '$scope', '
        $scope.emsCommonModel.emstype == "openstack" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "scvmm" && $scope.emsCommonModel.default_hostname ||
        $scope.emsCommonModel.emstype == "openstack_infra" && $scope.emsCommonModel.default_hostname ||
+       $scope.emsCommonModel.emstype == "nuage_network"  && $scope.emsCommonModel.default_hostname ||
```

Comment:
Remove the `$scope.emsCommonModel.default_hostname` condition here, since it appears that there is no hostname field in the form. This incorrect condition also prevents the Validate button from being enabled.